### ISSUE TITLE
Internal server error on production for sched_c

### ISSUE THREAD

**Issue by jontours** [18F/openFEC#1950]
Was verifying load times on sched_c this morning and ran across a server error on this query:

https://api.open.fec.gov/v1/schedules/schedule_c/?api_key=DEMO_KEY&per_page=20&sort=loan_balance&page=1&sort_hide_null=true

It goes away if you add a minus sign in front of loan_balance, which tells me that a null is being referenced somewhere.

**Comment by jontours** [comment]
This is probably an issue with parsing the `image_num` field to generate the `pdf_url`.  Will be resolved once the release is merged into master.

**Comment by noahmanger** [comment]
Is this still an issue?

**Comment by jontours** [comment]
Nope it's resolved.
